postgrest/_async/client.py

```diff
@@ -49,7 +51,13 @@ def create_session(
         timeout: Union[int, float, Timeout],
         verify: bool = True,
         proxy: Optional[str] = None,
+        http_client: Optional[AsyncClient] = None,
     ) -> AsyncClient:
+        if http_client is not None:
+            http_client.base_url = base_url
+            http_client.headers = headers
```

Comment:
should also pass `timeout`, `verify`, `proxy`, here?

Reply:
I didn't pass those as I don't want them to be override the `httpx` client the developer will pass. The only options we need to maintain are the `base_url` and `headers`. The others should be manually configured by the developer if they are passing their own `httpx` client.

Reply:
got it, lets at least deprecate the usage of the other params in favor of the httpx client as in https://github.com/supabase/functions-py/pull/201/files#r2083278694